Dockerfile

```diff
@@ -15,6 +17,11 @@ LABEL io.k8s.description="ASP.NET 5.0" \
 
 RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
```

Comment:
@sabre1041 Thanks for using a single `RUN` instruction. Can you combine the others as well?

@alberttwong FYI, using multiple RUN instructions results in Docker creating extra layers, so it's better practice to use a multi-line RUN statement as Andrew has done for his changes.